packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-customer-strategy.ts

```diff
@@ -80,8 +82,10 @@ export default class BraintreeAcceleratedCheckoutCustomerStrategy implements Cus
                 checkoutPaymentMethodExecuted();
             }
 
-            if (shouldRunAuthenticationFlow) {
-                await this.braintreeAcceleratedCheckoutUtils.runPayPalConnectAuthenticationFlowOrThrow();
+            if (shouldRunAuthenticationFlow && !this.isFastlaneEnabled) {
+                await this.braintreeFastlaneUtils.runPayPalConnectAuthenticationFlowOrThrow();
+            } else {
+                await this.braintreeFastlaneUtils.runPayPalFastlaneAuthenticationFlowOrThrow();
```

Comment:
Should we run Fastlane Authentication flow if `shouldRunAuthenticationFlow` is `false`?

Reply:
fixed condition